common/src/main/java/com/movie/recommender/common/model/Movie.java

```diff
@@ -33,7 +33,59 @@ public class Movie {
     private double voteAverage;
     private int voteCount;
 
-    // Inner Classes for nested objects
+
+    @Data
+    public static class MovieList {
```

Comment:
move to separate files

/models
   Movie.java - Contains Movie, Genre, Collection, etc.
   MovieList.java - Contains MovieList and Dates
   MovieCredits.java - Contains MovieCredits and CastMember.
   MovieKeywords.java - Contains MovieKeywords and Keyword.